onlyswaps-verifier/src/eth.rs

```diff
@@ -0,0 +1,201 @@
+use crate::config::NetworkConfig;
+use crate::eth::IRouter::SwapRequestParameters;
+use crate::eth::Router::RouterInstance;
+use crate::parsing::TransferReceipt;
+use crate::pending::{RequestId, Verification, extract_pending_verifications};
+use crate::signing::{ChainService, VerifiedSwap};
+use alloy::network::EthereumWallet;
+use alloy::primitives::{Address, FixedBytes, U256};
+use alloy::providers::{DynProvider, Provider, ProviderBuilder, WsConnect};
+use alloy::signers::local::PrivateKeySigner;
+use alloy::sol;
+use anyhow::anyhow;
+use async_trait::async_trait;
+use futures::future::{try_join, try_join_all};
+use std::collections::HashMap;
+use std::hash::Hash;
+
+sol!(
+    #[allow(clippy::too_many_arguments)]
+    #[derive(Debug, PartialEq, Eq, Hash, PartialOrd, Ord)]
+    #[sol(rpc)]
+    Router,
+    "../onlysubs-solidity/out/Router.sol/Router.json"
+);
+
+sol!(
+    #[allow(clippy::too_many_arguments)]
+    #[derive(Debug, PartialEq, Eq, Hash, PartialOrd, Ord)]
+    #[sol(rpc)]
+    ERC20FaucetToken,
+    "../onlysubs-solidity/out/ERC20FaucetToken.sol/ERC20FaucetToken.json"
+);
```

Comment:
Why do you need the faucet here??

Reply:
true - it only needs to be `ERC20` here - will update